whois-api/src/main/java/net/ripe/db/whois/api/httpserver/dos/IpBlockListFilter.java

```diff
@@ -0,0 +1,65 @@
+package net.ripe.db.whois.api.httpserver.dos;
+
```

Comment:
It is not a part of dos filter , can you create new package or move to a different one?

Reply:
oh...true
I am moving to httpserver package

Reply:
Done